## Section 2: Central limit Theorem Analytics Challenge:
## Part 1
#### You are a Business Analyst working for a logistics delivery company. 
#### A business client has requested a large freight to be transported urgently from Denver to Salt Lake City. When asked about the weight of the cargo, they could not supply the exact weight. However, they have specified that there are a total of 36 boxes.
#### From prior experience with this client, you know that this type of cargo follows a distribution with a mean if u = 72 lbs (32.66 kg) and a standard deviation of s = 3 lbs (1.36 kg). 
#### The only plane you currently have at Denver is a Cessna 209B Grand Caravan with a max cargo weight of 2,630 lbs (1,193 kg).
#### Based on this information, what is the probability that all of the cargo can be safely loaded onto the plane and be transported? 

In [1]:
#Import dependancies 
import numpy as np

In [9]:
#define variables
u = 72
sd = 3
n = 36
plane_capacity = 2630

In [8]:
#Calculate standard deviation of the mean
sd_mean = sd / np.sqrt(n)
print(sd_mean)

0.5


In [12]:
#Calculate the critical mass for boxes
box_cm = round(plane_capacity / n,2)
print(f"{box_cm} lbs per box")

73.06 lbs per box


In [15]:
#Calculate Z score
z_score = round((box_cm - u) / sd_mean,2)
print(f"{z_score} standard deviations away from the center")

2.12 standard deviations away from the center


In [16]:
# Using http://www.z-table.com/
p = 0.9830

The probability that the plane can safely take off is 98.30%. The data show that the average mass of a box is going to be less than 73.06lbs per box, the critical mass of a box for this plane.

## Part 2
#### You are a Quantitative Analyst working for a Wall-Street investment fund.
#### The team of traders, under your supervision, earns profits which can be approximated with a Laplace distribution. Profits have a mean of 95.70 and a standard deviation of 1,247. Your team makes about 100 trades every week. 
#### Questions
- What is the probability of your team making a loss in any given week?
- What is the probability of your team making over 20,000 in any given week?

In [21]:
#define variables 
mean = 95.70
stdev = 1247
trades_per_week = 100
target = 20000

In [22]:
#Calculate standard deviation of the mean
sd_of_mean = round(stdev / (np.sqrt(trades_per_week)),2)
print(sd_of_mean)

124.7


In [26]:
#Calculate X-critical 
x_crit = round(target / trades_per_week,2)
print(f"${x_crit} per trade")

$200.0 per trade


In [28]:
#Calculate Z score
trade_z_score = round((x_crit - mean) / sd_of_mean,2)
print(trade_z_score)

0.84


In [37]:
# Using http://www.z-table.com/
trade_p = 0.7995

In [34]:
#Calculate chance of making money (going right on the distribution chart)
chance = round(1 - trade_p,2) *100
print(f"{chance}% chance of making ${target} in a given week")

20.0% chance of making $20000 in a given week


In [35]:
#Calculate chance of making a loss
#set new x_crit of 0
x_crit_0 = 0
print(f"${x_crit_0} per trade")

$0 per trade


In [36]:
#Calculate Z score
trade_z_score_0 = round((x_crit_0 - mean)/sd_of_mean,2)
print(trade_z_score_0)

-0.77


In [39]:
# Using http://www.z-table.com/
trade_p_0 = 0.2206
print(f"{trade_p_0*100}% chance of making ${x_crit_0} in a given week")

22.06% chance of making $0 in a given week


There is a 22.06% chance of a loss in a given week, 20.05% of making $20,000 in a given week, and 57.89% chance that something else will happen in a given week. 